src/box/iproto_constants.h

```diff
@@ -164,6 +164,7 @@ extern const char *iproto_flag_bit_strs[];
 	 */								\
 	_(SQL_INFO, 0x42, MP_MAP)					\
 	_(STMT_ID, 0x43, MP_UINT)					\
+	_(IS_SYNC, 0x49, MP_BOOL)					\
```

Comment:
Let's better leave the `0x40-0x4f` range for all the SQL keys.
You may make IS_SYNC = 0x60, for example.

Reply:
Fixed.